packages/pass-style/src/types.d.ts

```diff
@@ -31,7 +31,7 @@ export type PassStyle =
   | 'error'
   | 'promise';
 
-export type TaggedOrRemotable = 'tagged' | 'remotable';
+export type TaggedOrRemotable = 'tagged' | 'selector' | 'remotable';
```

Comment:
Selector definitely does not belong in this union.

Reply:
its a container for a string! yeah ok was mostly just following 'tagged' around the code base